Src/PCompiler/CompilerCore/CompilerConfiguration.cs

```diff
@@ -154,6 +158,10 @@ public CompilerConfiguration(ICompilerOutput output, DirectoryInfo outputDir, IL
         /// Gets or sets a value indicating whether debug information should be included in output.
         /// </summary>
         public bool Debug { get; set; }
+        public int Timeout { get; set; }
```

Comment:
Can we pull these configuration into a separate class and make then `PVerifierConfig`